Makefile

```diff
@@ -25,3 +26,13 @@ pygments:
 
 build:
 	pip install -Ue .[docs,test]
+
+
+clean:
+	rm -fr $(ROOT).eggs/
+	rm -fr $(ROOT)build/
+	rm -fr $(ROOT)dist/
+	rm -fr $(ROOT)docs/_build/
+	find $(ROOT)edb/server -path pgproto -prune -name "*.c" -delete
```

Comment:
Maybe we use `git clean` that would take care of relevant c files automatically?

Reply:
`git clean -X` sounds like a good plan, let me try this. I didn't know about it 🤦🏻‍♂️